src/main.cpp

```diff
@@ -426,6 +429,10 @@ int main(int argc, char* argv[])
     const QStringList cliProfiles = parser.values(profileToOpen);
     const QStringList onlyProfiles = parser.values(onlyPredefinedProfileToShow);
 
+    const bool steamModestatus = parser.isSet(steamMode);
+    QSettings settings;
+    settings.setValue("steamMode", steamModestatus);
```

Comment:
Hmm, we use settings for things that should persist between Mudlet launches - and the steam mode isn't one of them.

It would be better to define a new public variable in mudlet.h akin to `smMirrorToStdOut` that tracks if steam mode is enabled, and you can set it just like we do a bit below - `mudlet::self()->smMirrorToStdOut =`

Reply:
I did not realize those were for persistent settings, thanks for letting me know. I've implemented the suggested feedback. I really appreciate you giving examples so I know that I'm on the right track. 🙏

Reply:
@vadi2 IIRC since the `mudlet` instance is a static singleton (isn't it?) can we not use the simpler `mudlet::smMirrorToStdOut` (in that example) - i.e. literally replacing any `mudlet::self()->` with `mudlet::` to refer to any ***static*** members?